### PR TITLE
event: Add Key

### PR DESCRIPTION
**Pull request by hajimehoshi**
This also renames event names from `Key*` to `KeyboardKey*`.

Updates #926

@beoran , please take a look.

### CODE REVIEW

File: event/event.go [hunk 1]
```diff
@@ -25,32 +25,30 @@ package event
...
 	Character rune
 }
 
-// KeyDown is an event that occurs when a key is pressed on the keyboard.
-type KeyDown struct {
-	// Code is the key code of the key pressed or released.
-	// TODO: this should change later from an int to an enumeration type.
-	Code int
+// KeyboardKeyDown is an event that occurs when a key is pressed on the keyboard.
+type KeyboardKeyDown struct {
```
Comment (beoran) [inline event/event.go]: The name KetboardKeyDown is longer, but consistent with names like MouseButtonDown, so I would say this is an improvement after all.

File: event/keys.go [hunk 2]
```diff
@@ -0,0 +1,129 @@
+// Copyright 2013 The Ebiten Authors
```
Comment (beoran) [inline event/keys.go]: Yes, this file also looks fine.

File: genkeys.go [hunk 3]
```diff
@@ -306,6 +306,24 @@ const (
 )
 `
 
+const eventKeysTmpl = `{{.License}}
```
Comment (beoran) [inline genkeys.go]: This generator code also looks OK.

**Comment by hajimehoshi** [comment]
Thank you for your reviewing!

**Pull request merged**